.github/workflows/release.yml

```diff
@@ -243,8 +243,6 @@ jobs:
       run: |
         git config user.email 'kiali-dev@googlegroups.com'
 
-        git config user.name 'kiali-bot'
```

Comment:
What would happen to the workflows now that git is not configured with a user.name? We need to know if GitHub Actions already define a generic username or if a git config user.name can be left undefined.

Reply:
If it is OK to remove user.name config, then additional PRs need to be submitted for the other repos that have similar workflows with similar lines:
* https://github.com/kiali/kiali-operator/blob/72cee3f7903990ef26cfaba5e4454c51561fc9f7/.github/workflows/release.yml#L207
* https://github.com/kiali/kiali.io/blob/c5ff023a7528f49a0cd3280c70e4f97178efbbf6/.github/workflows/release.yaml#L113
* https://github.com/kiali/helm-charts/blob/544472b3e247801602ba49eaf01601e0e58e25d0/.github/workflows/release.yml#L168
* https://github.com/kiali/openshift-servicemesh-plugin/blob/033a693b4b5be5cb39ae73788dff2e8844df73d8/.github/workflows/release.yaml#L234

Reply:
Sorry for the late reply.

 Apparently, user.name config is necessary to push to git usually. Otherwise the terminal would prompt an error to fill it in. But as kiali-bot is not in use, it'll never be staging or pushing to git. Seems it's safe to remove. 

I only just started with opensource, so, please correct me if I'm wrong or approaching the problem poorly. Thanks.